dd-trace/src/telemetry.rs

```diff
@@ -84,11 +89,12 @@ macro_rules! telemetry_metrics {
 }
 
 telemetry_metrics!(
-    SpansCreated => ("spans_created", Tracers, Count),
-    SpansFinished => ("spans_dropped", Tracers, Count),
-    SpansEnqueuedForSerialization => ("spans_enqueued_for_serialization", Tracers, Count),
-    TraceSegmentsCreated => ("trace_chunks_created", Tracers, Count),
-    TraceSegmentsClosed => ("trace_chunks_closed", Tracers, Count),
+    SpansCreated => ("spans_created", Tracers, Count, []),
+    SpansFinished => ("spans_finished", Tracers, Count, []),
+    SpansEnqueuedForSerialization => ("spans_enqueued_for_serialization", Tracers, Count, []),
+    SpansDroppedBufferFull => ("spans_dropped", Tracers, Count, ["reason" => "overfull_buffer"]),
+    TraceSegmentsCreated => ("trace_chunks_created", Tracers, Count, []),
+    TraceSegmentsClosed => ("trace_chunks_closed", Tracers, Count, []),
```

Comment:
Are `trace_chunks_created` and `trace_chunks_closed` metrics defined in https://github.com/DataDog/dd-go/blob/prod/trace/apps/tracer-telemetry-intake/telemetry-metrics/static/common_metrics.json?

(`trace_segments_created` and `trace_segments_closed` are)